src/App.vue

```diff
@@ -29,6 +29,15 @@ export default class App extends Vue {
     private isRequestLoaded = false;
 
     public async created() {
+        // Register service worker if necessary (and possible).
+        if ('serviceWorker' in navigator) {
+            navigator.serviceWorker.register('./ServiceWorker.js', {
+                scope: './',
```

Comment:
Keep in mind, that the `App.vue` component _always_ runs, also when the entry point is _not_ the root URL. All entry points that _we_ create are calling the root URL _currently_, but we cannot say that this will be the case for all user-created and future cases.

Is it thus possible, to set the scope to `'/'` instead?

Reply:
Should be possible, as long as the ServiceWorker is located in root, yes.